visualize/static/scripts/visualizations/new_view.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
+
+var NewView = DefaultView.extend({
```

Comment:
What is this view for? Is it just a test, "hello world" type thing? If so, it probably doesn't need to be pushed up (unless we want to use it to show people how to create visualizations?)